### ISSUE TITLE
I want to know the hyperparameter settings when training SenseVoice-s

### ISSUE THREAD

**Issue by zte-tcb** [FunAudioLLM/SenseVoice#149]
## ❓ Questions and Help
Hello, I would like to know the hyperparameter settings when training SenseVoice-s, such as batch size, optimizer, lr or warmup, etc.

**Comment by zte-tcb** [comment]
I see these in weight config
